### ISSUE TITLE
Make frontend updates in "Create Project" / "Edit Project" flow

### ISSUE THREAD

**Issue by michellema1208** [gitcoinco/grants-hub#299]
**Create & Edit Project flows** (same changes apply to both flows)

- [x] Mark required vs. optional field (see screenshots below)
- [x] Add form error states (see screenshots below)
- [x] Update copy in the "save changes" modal to say lose not loose
- [x] "Description" on the preview page should be **bolded** and have same font size as description text
- [x] Date on preview page should say "Created: <Month> <Date>, <Year>" (see screenshots below)

Archived
- [ ] Add form loading states (see screenshots below)

<img width="714" alt="Screen Shot 2022-09-07 at 10 45 20 PM" src="https://user-images.githubusercontent.com/26017180/189022213-c8bc9a44-16cf-47ec-bc51-4a6c10a03a53.png">

<img width="359" alt="Screen Shot 2022-09-07 at 10 48 14 PM" src="https://user-images.githubusercontent.com/26017180/189022585-335657a9-65bd-4e7c-9262-fc629576944f.png">

<img width="358" alt="Screen Shot 2022-09-07 at 10 48 55 PM" src="https://user-images.githubusercontent.com/26017180/189022700-754e9c88-c33b-47cf-9cee-de9a630b712d.png">

<img width="705" alt="Screen Shot 2022-09-07 at 11 02 52 PM" src="https://user-images.githubusercontent.com/26017180/189024621-cd9b58fe-c7f8-42ab-ae2c-4f6240ad790a.png">

**Comment by DanieleSalatti** [comment]
Required vs Optional: pending #315 - asked @codenamejason to refactor code so I can piggyback on his work

**Comment by DanieleSalatti** [comment]
>  Add form loading states (see screenshots below)

@michellema1208: the form loads instantly, I don't think we need this?

**Comment by michellema1208** [comment]
@DanieleSalatti I shared this on Discord, but I found some formatting / duplication issues on the Project Preview & Project Details page! Discord msg: https://discord.com/channels/562828676480237578/962026258273804398/1021814338538983535 

Reposting here: 
1. The project profile page and project preview page has two “created on” timestamps AND the spacing for created & last edited fields are off // cc: @DanieleSalatti
Instructions to fix: 
a. Remove the "created on" timestamp in the left column and keep the "last edited" + "created" timestamps in the right column 
b. There should be a space between icon and "created on"/"last edited", and there should be a space removed between colon and the actual timestamp) 

![Image](https://user-images.githubusercontent.com/26017180/191376174-8edb6d96-8927-46d9-9755-5c565657eb27.png)

Given these bugs, moving this ticket back to WIP
